<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>visionLab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="recordGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="videoCanvas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>571</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>显示区域</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="calibrateButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>490</y>
       <width>113</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>校准</string>
     </property>
     <property name="icon">
      <iconset resource="design.qrc">
       <normaloff>:/icons/calibrate.png</normaloff>:/icons/calibrate.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>113</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
     <property name="icon">
      <iconset resource="design.qrc">
       <normaloff>:/icons/eye.png</normaloff>:/icons/eye.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="recordButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>490</y>
       <width>113</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>跟踪</string>
     </property>
     <property name="icon">
      <iconset resource="design.qrc">
       <normaloff>:/icons/record.png</normaloff>:/icons/record.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="trackingComboBox">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>20</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="trackingLabel">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>跟踪算法</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="calibrationComboBox">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>80</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="calibrateLabel">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>90</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>校正方法</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="design.qrc"/>
 </resources>
 <connections/>
</ui>
